.gitlab-ci-check-apidocs.yml

```
@@ -32,3 +32,54 @@ test:apidocs:
     - swagger-cli validate docs/*.yml
     # Verify that the Swagger docs follow the autodeployment requirements
     - python3 verify_docs.py `find docs -name "*.yml"`
+
+trigger:apidocs:rebuild-mender-api-docs:
+  stage: .post
+  only:
+    changes:
+      - docs/*
+    refs:
+      - /^(master|staging|[0-9]+\.[0-9]+\.x)$/
+  image: alpine
+  before_script:
+    - apk add git python3 py3-pip
+    - pip3 install pyyaml
+    - git clone https://github.com/mendersoftware/integration.git mender-integration
+    - alias release_tool=$(realpath mender-integration/extra/release_tool.py)
+    - apk add --no-cache curl
+  script:
+    # If the repo is not recognized, ignore
+    - if ! echo $(release_tool --list git --all) | grep $CI_PROJECT_NAME; then
+    -  echo "Repository $CI_PROJECT_NAME not found in release_tool. Exiting"
+    -  exit 0
+    - fi
+    # Get the current stable release (latest tag matching the N.N.0 pattern)
+    - CURRENT_STABLE=$(cd mender-integration && git tag | egrep -e '^[0-9]+\.[0-9]+\.0$' | sort -n -r | head -n1 | sed -e 's/\.0$/.x/g')
+    # Enable the build of themender-api-docs site versions where this branch is in use
+    - BUILD_MASTER="false"
+    - BUILD_HOSTED="false"
+    - BUILD_LATEST="false"
+    - BUILD_OLDSTABLE="false"
+    - for version in $(release_tool  --integration-versions-including ${CI_PROJECT_NAME} --version ${CI_COMMIT_REF_SLUG} | cut -d"/" -f2); do
+    -   if [ "${version}" == "master" ]; then
+    -     BUILD_MASTER="true"
+    -   elif [ "${version}" == "staging" ]; then
+    -     BUILD_HOSTED="true"
+    -   elif [ "${version}" == "${CURRENT_STABLE}" ]; then
+    -     BUILD_LATEST="true"
+    -   else
+    -     BUILD_OLDSTABLE="true"
+    -   fi
+    - done
+    # Trigger the rebuild of mender-api-docs, if at least one of the build is enabled
+    - if [ "${BUILD_MASTER}" != "false" ] || [ "${BUILD_HOSTED}" != "false" ] || [ "${BUILD_LATEST}" != "false" ] || [ "${BUILD_OLDSTABLE}" != "false" ]; then
+    -   echo "Triggering mender-api-docs rebuild (master ${BUILD_MASTER}, hosted ${BUILD_HOSTED}, latest ${BUILD_LATEST}, oldstable ${BUILD_OLDSTABLE})"
+    -   curl -v -f -X POST
+          -F token=${SAAS_TRIGGER_TOKEN}
```

We need a different token. I suggest same name as repo:
```suggestion
          -F token=${MENDER_API_DOCS_TRIGGER_TOKEN}
```

(I can help setting up the token)

---

Ack